src/backend/distributed/commands/foreign_constraint.c

```diff
@@ -74,11 +74,11 @@ ConstraintIsAForeignKeyToReferenceTable(char *constraintName, Oid relationId)
 			continue;
 		}
 
-		Oid referencedTableId = constraintForm->confrelid;
+		Oid referencedRelationOid = constraintForm->confrelid;
```

Comment:
Why did you prefer relation over table here?

Reply:
it looks like we in general prefer relation over table around foreign keys